MediaBrowser.Model/Extensions/StringHelper.cs

```diff
@@ -22,11 +22,6 @@ public static string FirstToUpper(string str)
                 return str;
             }
 
-#if NETSTANDARD2_0
```

Comment:
This was needed for the UWP app IIRC

Reply:
With the move to .net5 we can no longer target .netstandard2.0
```
  Jellyfin.Data.csproj: [NU1202] Package Microsoft.EntityFrameworkCore.Relational 5.0.0 is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Package Microsoft.EntityFrameworkCore.Relational 5.0.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)
  Jellyfin.Data.csproj: [NU1202] Package Microsoft.EntityFrameworkCore.Sqlite 5.0.0 is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Package Microsoft.EntityFrameworkCore.Sqlite 5.0.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)
  Jellyfin.Data.csproj: [NU1202] Package Microsoft.EntityFrameworkCore.Relational 5.0.0 is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Package Microsoft.EntityFrameworkCore.Relational 5.0.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)
  Jellyfin.Data.csproj: [NU1202] Package Microsoft.EntityFrameworkCore.Sqlite 5.0.0 is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Package Microsoft.EntityFrameworkCore.Sqlite 5.0.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)
```

Reply:
It's mostly that EF Core 5 uses netstandard 2.1.